applications/registration-api/src/main/java/no/dcat/controller/ApiRegistrationPublicController.java

```diff
@@ -14,7 +16,7 @@
 import static org.springframework.web.bind.annotation.RequestMethod.GET;
 
 @RestController
-@RequestMapping(value = "/catalogs/{catalogId}/apis")
+@RequestMapping(value = "/catalogs")
```

Comment:
det sikkert virker, men det er ik generelt ikke god idé å blande namespace, hvis mulig.
/catalogs/{id} har nå clash med catalogs/publishedapis

det skulle være /publishedapis eller under noe ant root

Reply:
OK, men da må det også åpnes for det i nginx. Da vil jeg foreslå å ha stien published/apis. Da kan vi ha published/concepts etc

Reply:
vi kan jo ha /catalogs/{}/apis/{} - org-permission-checked
vi kan jo ha /public/apis/** - no-permission-requires, root er list